users/urls.py

```diff
@@ -5,4 +5,6 @@
     path('profile/', views.profile, name='users-profile'),
     path('login/', views.login, name='users-login'),
     path('logout/', views.logout, name='users-logout'),
+    path('edit_profile/', views.edit_profile, name='users-edit_profile')
```

Comment:
Please use [hyphens](https://pyphilly.org/hyphen-and-underscores-when-use-each-django-and-python/) instead of underscore in url 
```suggestion
    path('edit-profile/', views.edit_profile, name='users-edit-profile')
```